_posts/en/newsletters/2023-03-29-newsletter.md

```diff
@@ -0,0 +1,377 @@
+  forward payments even when they share a channel with a node that's
+  currently unavailable (such as the user of a mobile LN wallet).  This
+  requires the use of multi-party channels, which compose well with a
+  channel factory design he's previously described.  He also restates a
+  known benefit of [channel factories][topic channel factories] of
+  allowing channels to be rebalanced offchain, which may also allow
+  better use of capital.  He describes how to use both benefits in the
+  context of his previous innovation of the _tunable penalty_ layer for
+  LN.  We'll summarize tunable penalties, show how they can be used for
+  multiparty channels and channel factories, and then explain Law's new
+  results in context.
+
+    Alice and Bob create (but do not immediately sign) a transaction
+    which spends 50M sats from each of them (100M total) to a _funding
+    output_ which will require cooperation from both of them to spend.
+    In the diagrams below, we show confirmed transactions as shaded.
+
+    {:.center}
+    ![Alice and Bob create the funding transaction](/img/posts/2023-03-tunable-funding.dot.png)
+
+    They each also use a different output they individually control to
+    create (but not broadcast) two _state transactions_, one for each of
+    them.  The first output of each state transaction pays a trivial amount
+    (say 1,000 sat) to a _commitment transaction_.  A relative timelock
+    prevents each commitment transaction from being eligible for
+    confirmation onchain until a certain amount of time after its parent
+    state transaction is confirmed onchain.  Each of the two commitment
+    transactions is also funded by conflicting spends of the funding output
+    (meaning that only one of the commitment transactions can eventually be
+    confirmed).  With all the child transactions created, the transaction
+    which creates the funding output can be signed and broadcast.
+
+    {:.center}
+    ![Alice and Bob create their commitment transactions](/img/posts/2023-03-tunable-commitment.dot.png)
+
+    Each of the commitment transactions pays to the current state of the
+    channel.  For the initial state (_S<sub>0</sub>_), 50M sats is refunded
+    to each Alice and Bob (for simplicity we ignore transaction fees).
+    Alice or Bob can start the process of unilaterally closing the
+    channel by publishing their version of the state transaction; after
+    the enforced delay, they can then publish the corresponding
+    commitment transaction.  For example, Alice publishes her state
+    transaction and her commitment transaction (which pays both her and
+    Bob); at that point, Bob can simply never spend his state
+    transaction and instead spend the money used to create it at any
+    later time however he would like.
+
+    {:.center}
+    ![Alice spends honestly from the channel](/img/posts/2023-03-tunable-honest-spend.dot.png)
```

Comment:
I'm pushing an edit for this as a separate commit in case you don't like it and want to revert before publication (otherwise it can be squashed like normal).

Reply:
Dave, thanks I think the details adds clarity (even though also crowding)